manifests/init.pp

```diff
@@ -165,7 +165,7 @@
 
   case $facts['os']['family'] {
     'RedHat': {
-      if $facts['os']['release']['major'] =~ /(7|8)/ {
+      if Integer.new($facts['os']['release']['major']) >= 7 {
```

Comment:
We've dropped EL < 7 support so it can just be plain true.

That also means the RedHat sysvinit support around line 206 can be dropped, including the template.

Reply:
Ack, done